docs/modules/serialization/pages/serialization.adoc

```diff
@@ -1,32 +1,73 @@
 = Serialization
+:description: To be able to send data over a network between cluster members and/or clients, you have to serialize it into raw bytes. Hazelcast has many serialization methods to choose from, depending on which Hazelcast topology you are using and what you plan on doing with your data.
 
-Hazelcast needs to serialize the Java objects that you put into Hazelcast because
-Hazelcast is a distributed system. The data and its replicas are
-stored in different partitions on multiple cluster members.
-The data you need may not be present on the local member, and in that case,
-Hazelcast retrieves that data from another member. This requires serialization.
+{description}
 
-Serialization is used in the following cases:
+== Serialization in Client/Server Mode
 
-* Adding key/value objects to a map
-* Putting items in a queue/set/list
-* Sending a runnable using an executor service
-* Processing an entry within a map
-* Locking an object
-* Sending a message to a topic
+In xref:overview:topology.adoc[client/server topologies], you can serialize your data on the client side. For example, you can serialize data in JSON and add it to a map. This option is useful if you plan on using Hazelcast to store your data in binary. Clients can handle serialization without the members needing to know how to do so.
 
-Hazelcast optimizes the serialization for the basic types and their array types
-with its built-in serializers. By default, you cannot override this behavior; however,
-Hazelcast offers a way to override them using a configuration option,
-see the xref:overriding-built-in-serializers.adoc[Overriding the Default Serializers section].
-
-Hazelcast's built-in serializers can handle the following types:
+However, in the following cases, you must configure your serialization method on the member as well:

```

Comment:
> you must configure your serialization method on the member as well

As a newbie, it's a bit confusing that "member" means a node in a cluster since we mention client-server node. Do we have a glossary to link to? Or perhaps use help popups `<abbr title="Node in a cluster">member</abbr>`

Reply:
We have a small glossary, yes 😄 

I think we can link to the definition but also say `member (server)' to avoid users leaving the page just to find this out.